packages/fluentui/docs/src/examples/components/Toolbar/Content/ToolbarExampleCustomContent.shorthand.tsx

```diff
@@ -1,7 +1,42 @@
 import * as React from 'react';
-import { Button, Text, Toolbar } from '@fluentui/react-northstar';
+import { Button, Text, Toolbar, SplitButton } from '@fluentui/react-northstar';
 import { BoldIcon } from '@fluentui/react-icons-northstar';
 
+const ToolbarSplitButton = () => (
+  <>
+    <SplitButton
+      menu={[
```

Comment:
When navigation using keyboard and selecting menu item with Enter, the focus gets lost, do we want to handle it in this example?

Reply:
good point. it is expected, I will double check if we have it in the best practices for the split button. no need to put it in the example I think